browser/permissions/permission_request_manager_browsertest.cc

```diff
@@ -0,0 +1,847 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
do we really need this whole thing?

Reply:
we need test, we don't need chromium copyright.

Reply:
nope. removed.